test/test_io.py

```diff
@@ -19,12 +19,44 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
+from contextlib import contextmanager
 import io
 import unittest
 import tempfile
+import fastavro
 from lsst.alert.packet.io import retrieve_alerts
+from lsst.alert.packet.schema import Schema
```

Comment:
Pretty sure the original intention was that tests for `Schema` would go in `test_schema.py`. I don't think it' s a big deal, but this code would feel more at home there to me.

Reply:
That's fair. It's a little weird because there's a circular dependency between `io` and `schema`. I think you're really seeing that loop. I think I'll keep this here.